src/lgar.cxx

```diff
@@ -205,6 +205,8 @@ extern void InitFromConfigFile(string config_file, struct model_state *state)
     }
   }
 
+  // seek to beginning of input after searching for 'verbosity'
+  fp.seekg(0, fp.beg);
```

Comment:
probably a good idea to call `clear()` here to reset any potential error flags on the stream before continuing.  C++ 11 will clear the `eofbit` in `seekg` (older c++ standards do not,  however), and there could potentially be other error flags set from processing the stream, so if the intent is to "start over", clear expressly resets all error flags. Also, `seekg(0, fp.beg)` is equivalent to `seekg(0)`.

```suggestion
  fp.clear();
  fp.seekg(0, fp.beg);
```

Reply:
Sweet! I had the same question! Great catch. Just merged it in. Let squash and merge this